autogpt_platform/backend/backend/api/features/onboarding_dump/quality.py

```
@@ -0,0 +1,169 @@
+    try:
+        response = await asyncio.wait_for(
+            client.chat.completions.create(
+                model=_MODEL,
+                messages=[{"role": "user", "content": _PROMPT.format(transcript=text)}],
+                temperature=0,
+                max_tokens=100,
+            ),
+            timeout=_TIMEOUT_SECONDS,
+        )
+        content = response.choices[0].message.content or ""
+    except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)